docs/i18n.rst

```diff
@@ -148,16 +131,6 @@ Once you've reviewed the changes, submit them in a pull request for the
 ``l10n: update strings to be translated for vX.Y.Z`` pull request as a
 template.
 
-The new source strings will only be visible to translators in
-`Weblate`_ after they've been merged to ``securedrop/develop`` and
-that branch has been merged into ``securedrop-i18n/i18n``. The
-localization manager does this at the beginning of our release cycle.
-
-If this pull request is merged into ``develop`` after the ``release/X.Y.Z``
-branch has been cut, make sure to backport it, or you'll have to do so as part
-of the :ref:`release-day procedure <release_day>`, when it will be more
-confusing.
-
 .. _merge_develop_to_weblate:
 
 Merge develop into the Weblate fork
```

Comment:
A number of steps here can disappear entirely:
* Merge develop into the Weblate fork (and references in "Two weeks before the release: string freeze")
* Merge translations back to develop (and references in "Release day")
* Merge translations back to develop